osu.Game/Graphics/Containers/UserDimContainer.cs

```diff
@@ -37,7 +46,7 @@ public abstract class UserDimContainer : Container
 
         protected Bindable<bool> ShowVideo { get; private set; }
 
-        protected double DimLevel => EnableUserDim.Value ? UserDimLevel.Value : 0;
+        protected double DimLevel => DimOffset.Value + (EnableUserDim.Value ? UserDimLevel.Value : 0);
```

Comment:
This seems potentially unclamped too, from both sides this time. Probably best to just apply the clamp here and leave the other expressions as they are (or even remove the `Math.Min(1 - DimLevel, 1)` calls altogether?)